content/linked-terms.js

```diff
@@ -18,5 +18,9 @@ module.exports = [
   {
     matches: '.dvc/',
     url: '/doc/user-guide/project-structure/internal-files'
+  },
+  {
+    matches: 'dvc experiments',
+    url: '/doc/command-reference/exp'
   }
```

Comment:
Unrelated: Maybe we should apply something like this to `doctor` and any other aliases to avoid duplicating entries in the cmd ref.